src/main/java/application/ui/console/MoneyManagerConsoleAPP.java

```diff
@@ -0,0 +1,266 @@
+
+
+public class MoneyManagerConsoleAPP {
+    private final Engine engine = new Engine();
+    private static final String MENU =
+            """
+                            MENU
+                            1.Dodaj transakcje
+                            2.Przelew między kontami/portfelami (intertransakcja)
+                            3.Tworzenie nowego konta
+                            4.Tworzenie nowego portfela
+                            5.Wyswietlenie stanu konta/portfela
+                            6.Wyswietlenie pelnych danych konta/portfela
+                            7.Usuwanie konta/portfela
+                            8.Usuwanie danych uzytkownika
+                            9.Koniec programu
+                            Wybierz opcje:\s
+                    """;
+
+    public static void main(String[] args) {
+        new MoneyManagerConsoleAPP().work();
+    }
+
+    private void work() {
+        /* =================== PRZYWITANIE ======================== */
+
+        System.out.println("Witaj w programie \"MoneyManager\" !");
+        System.out.println("Kobieta jak kobieta, twór ma różne rządze, zerknij tutaj, może kradnie Ci pieniądze...");
+
+        /* ===================== PROGRAM ========================== */
+
+        /* ------------------ Opcjonalne ładowanie zrzuconej bazy danych ------------------ */
+
+        if(engine.preCheckDatabase()){
+            while (true) {
+                System.out.println("Wykryto zrzut bazy danych. Czy zaladowac go? [1.Tak/2.Nie]");
+                int choice = DataEnter.enterInt();
+                if (choice == 1) {
+                    try{
+                        engine.loadAll();
+                        System.out.println("Pomyslnie zaladowano poprzedni zrzut!");
+                        break;
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                        System.out.println("Nastąpił błąd! Wychodze z programu!");
+                        System.exit(-1);
+                    }
+                }
+                else if (choice == 2) {
+                    System.out.println("Przechodze do dalszej czesci programu, bez ladowania bazy!");
+                    break;
+                }
+                else {
+                    System.out.println("Brak takiej opcji! Dostępne opcje to 1 lub 2");
+                }
+            }
+        }
+
+        while (true) {
+            System.out.println(MENU);
+            int function = DataEnter.enterInt();
+            if (function == 1) {    /* ------------------ Dodawanie transakcji ------------------ */
+                if (engine.anyMoneyStorages()) {
+                    while (true) {
+                        System.out.println("1.Wplata czy 2.wyplata gotowki?[1/2]: ");
+                        int transactionTypeChoice = DataEnter.enterInt();
+                        if (transactionTypeChoice == 1 || transactionTypeChoice == 2) {
```

Comment:
nie żebym się czepiał, ale jak przypadkiem wejdziesz w tą opcję to nie musisz przejść przez cały proces, bez możliwości cofnięcia

Reply:
Nie wiem czy dobrze rozumiem. Chodzi o to że nie można się cofnąć po wybraniu opcji (brakuje exit)?

Reply:
tak, albo back
sorki, że dopiero teraz odp, ale coś mnie więziło w poniedziałek i byłem nie do życia